app/src/main/java/helper/CourseDataHandler.java

```diff
@@ -233,6 +233,13 @@ public void markAsRead(int moduleId) {
         realm.commitTransaction();
     }
 
+    public void markAsUnread(int moduleId) {
```

Comment:
* When you instantiate a realm db object you need to close it also, so please close it at the end of method.
* Use a single method, for marking read or unread, by passing in a boolean as parameter. You can name the method something like `markAsReadAndUnread` or come up with a short one.